RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -578,11 +605,11 @@ public void FindReferencesInScene(object obj)
 
         protected override void Update()
         {
-            if (_scrollTreeToSelected && _scrollTarget > 0)
+            if (_scrollTreeToSelected && _scrollTarget >= 0)
             {
                 _scrollTreeToSelected = false;
                 _treeScrollPosition.y = _scrollTarget;
-                _scrollTarget = 0;
+                _scrollTarget = -1f;
```

Comment:
As stated at top of the file, scollTarget 0 is valid is first item is selected, through mouse inspect or anything. Changed check to >= 0 and used -1f to determine not valid.